Video.js

```diff
@@ -81,6 +81,18 @@ export default class Video extends Component {
     return await NativeModules.VideoManager.save(options, findNodeHandle(this._root));
   }
 
+  pause = async () => {
+    await this.setPlayerPauseState(false);
+  };
+
+  play = async () => {
+    await this.setPlayerPauseState(true);
```

Comment:
sorry but I think naming is not correct here ...
When I call: setPlayerPauseState(true); I am expecting the playback to be paused... It is possible to invert the logic please ?

Reply:
Agree - fixed ✅